plotly/offline/offline.py

```diff
@@ -152,10 +177,12 @@ def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly',
                       graph descriptions.
 
     Keyword arguments:
-    show_link (default=True) -- display a link in the bottom-right corner of
-                                of the chart that will export the chart to
-                                Plotly Cloud or Plotly Enterprise
-    link_text (default='Export to plot.ly') -- the text of export link
+    config -- a dictionary with configuration attributes for the plot,
+        default contains the following two elements:
+        showLink (default=True) -- display a link in the bottom-right corner of
+                                    of the chart that will export the chart to
+                                    Plotly Cloud or Plotly Enterprise
+        linkText (default='Export to plot.ly') -- the text of export link
```

Comment:
we'll want to describe that these are `plotly.js` configuration options and provide a link to our documentation: https://plot.ly/javascript/configuration-options/